### PR TITLE
.coafile: Add YAMLLintBear support

### PR DESCRIPTION
**Pull request by akshkr**
This support ensures checking yml files using YAMLLintBear

Closes https://github.com/coala/coala-gedit/issues/19

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -8,3 +8,7 @@ allow_trailing_whitespace = False
 [autopep8]
 bears = PEP8Bear
 default_actions = PEP8Bear: ApplyPatchAction
+
+[ymlfiles]
+bears = YAMLLintBear
+files = **.yml
```
Comment (Makman2) [inline .coafile]: missing newline at EOF

**Comment by Makman2** [comment]
Commit message:
1. I think you can remove "support" from the shortlog. Just ".coafile: Add YAMLLintBear" is good imo :)
2. As you modified yml-files to adhere to the now enforced standards, I would recommend to describe this fact inside the commit body :)

**Review by khanchi97 — APPROVED** [review]
Changes LGTM :+1:

**Comment by khanchi97** [comment]
ack 6946042c

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**